pom.xml

```
@@ -61,6 +63,16 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+
+			<!-- CLOUD SDK -->
+			<dependency>
+            	<groupId>com.sap.cloud.sdk</groupId>
+        	    <artifactId>sdk-bom</artifactId>
```

I think this one should be preferred, as the other bom manages also third-party dependencies, which can lead to unexpected outcomes.

```suggestion
        	    <artifactId>sdk-modules-bom</artifactId>
```